lerobot/common/robot_devices/cameras/intelrealsense.py

```diff
@@ -149,6 +153,7 @@ class IntelRealSenseCameraConfig:
     color_mode: str = "rgb"
     use_depth: bool = False
     force_hardware_reset: bool = True
+    rotation: int | None = None
```

Comment:
COuld we add rotation on opencv.py to stay consistent?

Reply:
Done 62ddbb2cd683b1f392b8ecee577fce0b8ded91fb
(Had to do it anyway for the nav cam)